include/pacmod3_dbc_ros_api.h

```diff
@@ -79,9 +79,7 @@
 namespace pm_msgs = pacmod3_msgs;
 namespace cn_msgs = can_msgs;
 
-#endif  // ROS1
-
-#if ROS_VERSION==2
+#elif ROS_VERSION==2
```

Comment:
Does ROS has a special macro defined for this purpose? Or ROS_VERSION is the one?